low-level/json_msgs/schemas/actuators/SSPL-LL_Actuator_Request.json

```diff
@@ -73,7 +73,10 @@
 						"required": false
 					}
 				},
-
+				"target_node_id": {
```

Comment:
this is going to fail any new existing actuator requests, so double confirm with CSM again

Reply:
I have asked @pawankumarsrivastava to double check and he confirmed. I will create a ticket for the same.

Reply:
ok